.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
-    "env": {
-        "es6": true,
-        "browser": true
-    },
-    "extends": "eslint-config-digitalbazaar",
+  env: {
+    browser: true
+  },
+  plugins: ["jsdoc"],
```

Comment:
this produces error that strings must be single quoted in other files, but not here?  What's up with that?

Also, extra blank line at bottom.

Reply:
good question actually because veres-wallet's eslint does run on .eslint so not entirely sure. made the corrections.